examples/4_train_policy_with_script.md

```diff
@@ -0,0 +1,157 @@
+
+```bash
+python lerobot/scripts/train.py policy=diffusion env=pusht
+```
+
+If you want to train ACT with Aloha, you can do:
+
+```bash
+python lerobot/scripts/train.py policy=act env=aloha
+```
+
+**Notice, how the config overrides are passed** as `param_name=param_value`. This is the format the Hydra excepts for parsing the overrides.
+
+## Overriding configuration parameters in the CLI
+
+If you look in `env/aloha.yaml` you might see:
+
+```yaml
+# lerobot/configs/env/aloha.yaml
+env:
+  task: AlohaInsertion-v0
+```
+
+And if you look in `policy/act.yaml` you might see:
+
+```yaml
+# lerobot/configs/policy/act.yaml
+dataset_repo_id: lerobot/aloha_sim_insertion_human
+```
+
+But our Aloha environment actually supports a cube transfer task as well. To train for this task, you _could_ modify the two configuration files respectively.
+
+We need to select the cube transfer task for the ALOHA environment.
+
+```yaml
+# lerobot/configs/env/aloha.yaml
+env:
+   task: AlohaTransferCube-v0
+```
+
+We also need to use the cube transfer dataset.
+
+```yaml
+# lerobot/configs/policy/act.yaml
+dataset_repo_id: lerobot/aloha_sim_transfer_cube_human
+```
+
+Now you'd be able to run:
+
+```bash
+python lerobot/scripts/train.py policy=act env=aloha
+```
+
+and you'd be training and evaluating on the cube transfer task.
+
+OR, your could leave the configuration files in their original state and override the defaults via the command line:
+
+```bash
+python lerobot/scripts/train.py \
+    policy=act \
+    dataset_repo_id=lerobot/aloha_sim_transfer_cube_human \
+    env=aloha \
+    env.task=AlohaTransferCube-v0
+```
+
+There's something new here. Notice the `.` delimiter used to traverse the configuration hierarchy.
+
+Putting all that knowledge together, here's the command that was used to train https://huggingface.co/lerobot/act_aloha_sim_transfer_cube_human.
+
+```bash
+python lerobot/scripts/train.py \
+    hydra.run.dir=outputs/train/act_aloha_sim_transfer_cube_human \
+    device=cuda
+    env=aloha \
+    env.task=AlohaTransferCube-v0 \
+    dataset_repo_id=lerobot/aloha_sim_transfer_cube_human \
+    policy=act \
+    training.eval_freq=10000 \
+    training.log_freq=250 \
+    training.offline_steps=100000 \
+    training.save_model=true \
+    training.save_freq=25000 \
+    eval.n_episodes=50 \
+    eval.batch_size=50 \
+    wandb.enable=false \
+```
+
+There's one new thing here: `hydra.run.dir=outputs/train/act_aloha_sim_transfer_cube_human`, which specifies where to save the training output.
+
+---
+
+Now, why don't you try running:
+
+```bash
+python lerobot/scripts/train.py policy=act env=pusht dataset_repo_id=lerobot/pusht
+```
+
+That was a little mean of us, because if you did try running that code, you almost certainly got an exception of sorts. That's because there are aspects of the ACT configuration that are specific to the ALOHA environments, and here we have tried to use PushT.
```

Comment:
Not sure about this one ^^

Reply:
Please let me know which part you're referring to. Is it the informal tone in "that was a little mean of us"?

Reply:
If we do reference a command that doesnt work, we shouldnt say "you almost certainly got an exception of sorts.".
Instead, we expect to get a specific error. 
But I really not sure we should in this context.

Reply:
Maybe you could clarify that the policy config file contains values (dataset_repo_id) that are specific to the combination of policy and environment. If the environment config does not match what the policy configuration expects, an error is thrown.

Reply:
@Cadene I don't want to be explicit about the exception. This will almost certainly change, and IMO it adds no value to explain which one it is.

@dirkmcpherson I think that's a good suggestion.

Ptal